### DISCUSSION TITLE
[Bug]  Scoop won't update packages

### DISCUSSION THREAD

**Discussion by ChausseBenjamin** [ScoopInstaller/Scoop#5669] [Help and Questions]
#### Current Behavior
```
PS C:\Users\Benjamin\Documents\SongLink-DiscordBot> scoop update
Updating Scoop...
fatal: unable to access 'https://github.com/ScoopInstaller/Scoop/': getaddrinfo() thread failed to start
Update failed.
```
I tried rebooting the system and running it through a different powershell version. Didn't help...

#### Expected Behavior

Scoop should be fetching the mirrors normally.

#### Additional context/output

```
PS C:\Users\Benjamin\Documents\SongLink-DiscordBot> scoop checkup
No problems identified!
```

### System details

**Windows version:** 11

**OS architecture:** 64bit

**PowerShell version:** 7.3.7

#### Scoop Configuration

```json
{
  "last_update": "2023-09-26T04:06:53.5135158-04:00",
  "aria2-warning-enabled": false,
  "scoop_branch": "master",
  "scoop_repo": "https://github.com/ScoopInstaller/Scoop"
}

```

**Comment by rashil2000** [comment]
This looks like a problem with your Git installation. Are you able to use other Git repos in your machine normally?

**Comment by ChausseBenjamin** [comment]
Yup! I've been able to git clone (with both https and ssh) without any issues.

**Reply by rashil2000** [reply]
Can you go to ~\scoop\apps\scoop\current location and try doing a git pull there?
